.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parser":  "@typescript-eslint/parser",
+  "extends":  [
+    "plugin:@typescript-eslint/recommended"
```

Comment:
I'm fine with adding custom eslint rules if needed (please just list why in the PR), but this should extend from `@ky-is/eslint-config/typescript` and  remove the existing configuration in `package.json`.